consensus/polybft/state_transaction.go

```diff
@@ -0,0 +1,538 @@
+package polybft
```

Comment:
Please standardize all the errors in this file, there is no need to use `fmt.Errorf` everywhere